R/tm_a_pca.R

```diff
@@ -438,10 +438,10 @@ srv_a_pca <- function(id, data, reporter, filter_panel_api, dat, plot_height, pl
       selector_list = selector_list,
       datasets = data
     )
-
+    qenv <- teal.code::eval_code(data(), 'library("ggplot2")')
```

Comment:
Do we need `dplyr` and `tidyr` in `tm_a_pca`? Check out the `Show R Code`, it has some `dplyr` operations and `%>%`
```suggestion
    qenv <- teal.code::eval_code(data(), 'library("ggplot2");library("dplyr");library("tidyr")') # nolint quotes
```

Also added `# nolint` because you used `'` quotes for strings

<img width="938" alt="image" src="https://github.com/user-attachments/assets/39ad6d3e-3f96-4d48-893d-70f09f34c91a" />